lib/galaxy/jobs/runners/condor.py

```diff
@@ -177,6 +177,9 @@ def check_watched_items(self):
             galaxy_id_tag = cjs.job_wrapper.get_id_tag()
             try:
                 if os.stat(cjs.user_log).st_size == cjs.user_log_size:
+                    if cjs.job_wrapper.tool.tool_type == 'interactive':
+                        # If running, check for entry points...
+                        cjs.job_wrapper.check_for_entry_points()
```

Comment:
Would it work if you instead said `if cjs.job_wrapper.tool.tool_type != 'interactive' and os.stat(cjs.user_log).st_size == cjs.user_log_size` and then leave it the way it is now ? That way you'd call `check_for_entry_points ` only if condor reports the job as running (= `s1 and not (s4 or s7)`)